requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy>=1.9.2
+pandas>=0.16.2
+python-dateutil>=2.4.2
+pytz>=2015.4
+six>=1.9.0
```

Comment:
I think we should probably relax some of these requirements? for instance `numpy` and `pandas` don't really have to be the latest